composer.json

```diff
@@ -24,20 +24,19 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": ">=7.0.8",
```

Comment:
7.0 (and 7.1) is EOL, so we should probably start new projects with 7.2 (EOL: 30 Nov 2020) or we could even dare 7.3 maybe.

Reply:
I'm following Drupal core here. Of course we could enforce an higher version. But I would vote against blocking still supported versions.